src/Auth.php

```diff
@@ -156,15 +176,17 @@ protected static function get_signed_token( $user, $cap_check = true ) {
 			'exp'  => self::get_token_expiration(),
 			'data' => [
 				'user' => [
-					'id' => $user->data->ID,
+					'id'          => $user->data->ID,
+					'user_secret' => $secret,
```

Comment:
At the moment, this is the one thing that makes a refresh token different than an access token. Having this in both tokens makes the tokens identical.